pkg/convert/json_ng.go

```diff
@@ -0,0 +1,28 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+func FromNgJson(b []byte) (model.Messages, error) {
+	var msg model.Messages
+
+	err := json.Unmarshal(b, &msg)
+	if err != nil {
+		return model.Messages{}, fmt.Errorf("decode model.Messages to NG JSON: %w", err)
+	}
+
+	return msg, nil
+}
+
+func ToNgJson(m model.Messages) ([]byte, error) {
+	msg, err := json.Marshal(m)
```

Comment:
Same goes for here, structure that is returned, is not same as NG JSON structure